lib/ex_unit/lib/ex_unit.ex

```diff
@@ -299,7 +299,13 @@ defmodule ExUnit do
 
   defp put_seed(opts) do
     Keyword.put_new_lazy(opts, :seed, fn ->
-      :os.timestamp |> elem(2)
+      # We're using `rem System.system_time()` here
+      # instead of directly using :os.timestamp or using the
+      # :microsecond argument because the VM on Windows has... odd...
+      # precision. Calling with :microsecond will give us a multiple
+      # of 1000. Calling without it gives actual microsecond precision.
+      # Yeah, it's weird.
```

Comment:
Can we get rid of the various "..."s and the "Yeah it's weird" at the end? Trying to keep it professional 😄